openapi-cli/src/test/java/io/ballerina/openapi/generators/schema/ConstraintTests.java

```diff
@@ -118,13 +126,14 @@ public void testNestedArrayWithConstraint() throws IOException, BallerinaOpenApi
         assertFalse(hasErrors);
     }
 
-    @Test
+    @Test(description = "Tests with additional properties field has constraint.")
     public void testAdditionalPropertiesWithConstraint() throws IOException, BallerinaOpenApiException,
             FormatterException {
         OpenAPI openAPI = GeneratorUtils.normalizeOpenAPI(RES_DIR.resolve("swagger/constraint" +
                 "/additional_properties_with_constraint.yaml"), true);
         BallerinaTypesGenerator ballerinaSchemaGenerator = new BallerinaTypesGenerator(openAPI);
         SyntaxTree syntaxTree = ballerinaSchemaGenerator.generateSyntaxTree();
+//        TestUtils.deleteGeneratedFiles();
```

Comment:
Shall we remove this commented line ?

Reply:
Updated in https://github.com/ballerina-platform/openapi-tools/pull/1328/commits/1c1b409182ccd309a4da54dd57c18e65241dd1bc